client/tests/test-1.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('http://localhost:5173');
+});
```

Comment:
Line below. Try to run linter and formmater with permanent rules before pushing code.